main.py

```diff
@@ -1,10 +1,39 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player_name, player_info in data.items():
+        race, _ = Race.objects.get_or_create(
+            name=player_info["race"].get("name"),
+            description=player_info["race"].get("description"),
+        )
+
+        for skill in player_info["race"]["skills"]:
```

Comment:
Consider using `.get('skills', [])` to safely access the 'skills' key in `player_info['race']`. This will prevent potential `KeyError` exceptions if the key is missing.